jobtastic/__init__.py

```diff
@@ -14,4 +14,4 @@
 
 # -eof meta-
 
-from jobtastic.task import JobtasticTask
+from jobtastic.task import JobtasticTask  # NOQA
```

Comment:
Why import this if we aren't using it? Either drop the import or, more likely, add an `__all__` that includes this and everything else you want to be able to access by importing this file.